pkg/operator/ceph/csi/spec.go

```diff
@@ -601,6 +611,12 @@ func createCSIDriverInfo(ctx context.Context, clientset kubernetes.Interface, na
 	}
 
 	if apierrors.IsNotFound(err) {
+		if fsGroupPolicy != "" {
+			policy := k8scsi.FSGroupPolicy(fsGroupPolicy)
+			// FsGroupPolicy field is immutable, should be set only during
```

Comment:
So if they change this policy they will need to delete the CSI driver and restart the operator? How about a line about this in the csi doc ceph-csi-drivers.md at the same time as adding the link to the helm settings?

Reply:
@travisn we do have two options here. Either the user can delete the CSI driver object and restart the operator pod. or Rook itself can detect the changed configuration and delete the CSI driver object and recreate it. which one do you prefer?